Makefile

```diff
@@ -35,7 +48,11 @@ $(error No Git token found. Please provide it either via the GIT_TOKEN variable
 endif
 GIT_TOKEN ?= $(shell cat .git_token)
 endif
+
 LINUX_IMAGE ?= quay.io/orchestrator/ubi9-pipeline:latest
+JDK_IMAGE ?= registry.access.redhat.com/ubi9/openjdk-17:1.17
+BUILD_APPLICATION_SCRIPT ?= $(SCRIPTS_DIR)/build_application.sh
```

Comment:
maybe the application script should contain the name of the application (application id) so it would be generic here for any application (jira listener, m2k-kfunc,...), WDYT?

Reply:
we don't have a build script for each application: `build_application.sh` is the general purpose one designed to be reused by many applications. `m2k-func` is the only exception (for now) that needs its own code generation before the build.